.env.development

```
@@ -0,0 +1,50 @@
+# Development Environment - PRMCMS Consolidated Project
+# Project ID: flbwqsocnlvsuqgupbra (PRMCMS Main)
+# Region: us-east-2
+
+# Supabase Configuration
+SUPABASE_URL=https://flbwqsocnlvsuqgupbra.supabase.co
+SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImZsYndxc29jbmx2c3VxZ3VwYnJhIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTMzNzY2NzcsImV4cCI6MjA2ODk1MjY3N30.nhWOR862I7FsZ7oOxNnN5-Mp2C9gdhp2TXl84YEPwtw
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_ANON_KEY key should go before the SUPABASE_URL key

(UnorderedKey)